pootle/core/browser.py

```diff
@@ -85,14 +87,14 @@ def get_table_headings(choices):
     return filter(lambda x: x['id'] in choices, HEADING_CHOICES)
 
 
-def make_generic_item(path_obj):
+def make_generic_item(path_obj, **kwargs):
     """Template variables for each row in the table."""
     return {
         'href': path_obj.get_absolute_url(),
         'href_all': path_obj.get_translate_url(),
```

Comment:
@julen No. The only kwarg we are passing now is the priority sorting, and we don't want priority sorting when translating all the units.